manifests/base/kustomization.yaml

```diff
@@ -1,13 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-bases:
```

Comment:
Thank you for working on this feature @jutley! CI is not happy because of this change. We are still using Kustomize v2.0.0 . Can you please revert this change?

Reply:
Okay, I could see this being the issue! I'll get that installed and try again!

I'll also point out that I would prefer to just download the patch artifact as described, but the screenshot at https://argoproj.github.io/argo-cd/developer-guide/ci/ does not match how CircleCI looks for me. I just have the Test Summary, Configuration, Timing, and Parameters tabs. No Artifact tab.

Reply:
Ah, you are right. Circle CI is showing artifacts only to authenticated users. I could not find a way to enable it. Will change CI to just print the diff in logs.